docs/rdl/use-cases.md

```diff
@@ -1,29 +1,27 @@
 
-An **academic research team** needs to demonstrate the impact of a **risk analytics and urban planning project**, by making the data available for others to use. Their dataset are formatted according to the **RDL standards** and published in a data catalogue set up for the project, using the **template implementation** available through the RDL project.
 
-***Value***: The pre-designed open-source deployable solutions can assist a research group achieve impact efficiently.
+As an engineer in a geohazards risk consulting firm, Pete is in charge of delivering a package of final geospatial deliverables to their client, a multi-national development bank. Pete's team has just completed flood and earthquake risk assessments for seven Pacific Island nations. They have been asked to publish the loss data on the World Bank's Risk Data Library and a regional data catalog, and want to make sure potential users can find all the datasets relevant to each country analysis. They use the RDLS to create a metadata file and dataset for each of the seven countries, with each dataset including the hazard, exposure, vulnerability _and_ loss resources so users can find the full data package in future.
 
-______________________________________________________________________
 
-An **insurance industry analyst or model developer** needs to **build a new catastrophe model** or **analyse a new portfolio** to insure in an emerging market, where the company has limited experience and data. They are searching for existing data to support their analysis and find some existing data created by consultants in the development sector, shared via the Risk Data Library Collection of the World Bank Data Catalog, formatted according to the **RDL standards** and available to download and use.
 
-***Value***: The data can be used directly or improved to assess the risk to the new portfolio, and could expedite the placement of insurance cover for vulnerable communities.
+## Risk data users
 
-______________________________________________________________________
+As policy analyst in a humanitarian organisation, who is new to risk information, Tala is looking for a summary of what datasets are available with risk estimates in South East Asia. She executes a search for risk information and finds multiple data catalogs all using the RDLS. She is able to compile the metadata from those catalogs and produce her report on data availability and gap analysis more quickly than expected because all the datasets have consistent metadata that describes risk-specific information.
```

Comment:
```suggestion
As policy analyst in a humanitarian organisation, who is new to risk information, Tala is looking for a summary of what datasets are available with risk estimates for South East Asia. She executes a search for risk information and finds multiple data catalogs all using the RDLS. She is able to compile the metadata from those catalogs and produce her report on data availability and gap analysis more quickly than expected because all the datasets have consistent metadata that describes risk-specific information.
```